roles/filetree_create/templates/current_credentials.j2

```diff
@@ -3,19 +3,22 @@
 controller_credentials:
 {% endif %}
   - name: "{{ current_credentials_asset_value.name }}"
-    description: "{{ current_credentials_asset_value.description }}"
+    description: "{{ template_overrides_global.credential.description | default(current_credentials_asset_value.description) }}"
     credential_type: "{{ current_credentials_asset_value.summary_fields.credential_type.name }}"
-{% if current_credentials_asset_value.organization is defined and current_credentials_asset_value.organization is not none %}
+{% if current_credentials_asset_value.organization is defined and current_credentials_asset_value.organization is not none) %}
```

Comment:
There's an extra closing parenthesis at the end of the condition.

Reply:
I don't know why I didn't notice, but thank you very much :).